python/src/aac/validate/_validate.py

```diff
@@ -43,10 +44,13 @@ def validated_source(source: str) -> Generator[ValidationResult, None, None]:
 
 def _with_validation(user_definitions: list[Definition]) -> ValidationResult:
     result = _validate_definitions(user_definitions)
-    if result.is_valid:
-        return result
-    else:
-        raise ValidationError("Failed to validate content with errors:", *result.messages)
+    try:
```

Comment:
I couldn't leave it as a suggestion but I think this `try` should go around the `result = ...` part, too. I don't see any way in which this `if` statement would result in a `LanguageError`.

Reply:
Good catch, thanks!